src/Application/ApplicationManagement.tsx

```diff
@@ -142,88 +140,21 @@ class ManageApplicationContainer extends React.Component<
             gender: hackerDetails.gender,
             ethnicity: hackerDetails.ethnicity,
             needsBus: hackerDetails.needsBus,
-            github: hackerDetails.application.portfolioURL.github,
-            dropler: hackerDetails.application.portfolioURL.dropler,
-            linkedIn: hackerDetails.application.portfolioURL.linkedIn,
-            personal: hackerDetails.application.portfolioURL.personal,
-            other: hackerDetails.application.portfolioURL.other,
-            resumeFile: undefined,
-            jobInterest: hackerDetails.application.jobInterest,
-            skills: hackerDetails.application.skills,
-            essay: hackerDetails.application.essay,
-            comments: hackerDetails.application.comments,
+            application: hackerDetails.application,
             codeOfConduct_MCHACKS: hackerDetails.codeOfConduct,
             codeOfConduct_MLH: hackerDetails.codeOfConduct,
+            resumeFile: undefined,
           }}
           onSubmit={this.handleSubmit}
           render={this.renderFormik}
-          validationSchema={this.getValidationSchema(mode)}
+          validationSchema={getValidationSchema(
```

Comment:
Why was `this.` removed?

Reply:
I moved the validation schema into its own file and imported it.